# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/aop/hotfixes.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Ambassador-Of-Pain.git/aop.c.vanilla	2025-01-29 18:12:25.883333165 +0100
+++ Ambassador-Of-Pain.git/aop.c	2025-01-29 18:12:44.586666502 +0100
@@ -21,7 +21,7 @@
 	init_pair(1, COLOR_RED, COLOR_BLACK);
 
 start:	memset(field, ' ', 25*81);
-	snprintf(tmp, 96, "/usr/local/share/aop/aop-level-%02d.txt", level);
+	snprintf(tmp, 96, "/usr/share/aop/aop-level-%02d.txt", level);
 	f = fopen(argc>=2 ? argv[level] : tmp, "r");
 	if (!f) { endwin(); printf("Can't open level file.\n"); return 1; }
 	opx=ipx=px=3, opy=ipy=py=2; op=p; p += 700000 + level*373737;
